src/main/java/org/jabref/logic/importer/fetcher/MathSciNet.java

```diff
@@ -76,7 +76,7 @@ public URL getURLForEntry(BibEntry entry) throws URISyntaxException, MalformedUR
 
     @Override
     public URL getURLForQuery(QueryNode luceneQuery) throws URISyntaxException, MalformedURLException, FetcherException {
-        URIBuilder uriBuilder = new URIBuilder("https://mathscinet.ams.org/mathscinet/search/publications.html");
+        URIBuilder uriBuilder = new URIBuilder("https://mathscinet.ams.org/mathscinet/publications-search");
```

Comment:
Those should not be part of this PR

Reply:
Yes, I fixed it. It happened twice now, but I don't know why. Those changes were not displayed...